indico/modules/rb_new/client/js/modules/bookRoom/BookingSuggestion.jsx

```diff
@@ -0,0 +1,130 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
```

Comment:
2019 (we should run the update script at some point, but maybe we can wait with that until we do the license change as well?)